data/simpy.gram

```diff
@@ -326,7 +340,7 @@ star_targets: a=star_target (',' !'in' star_target)* [','] { a }
 star_target: '*' bitwise_or | atom t_tail+ | a=star_atom t_tail* { a }
 star_atom: a=NAME { store_name(p, a) } | '(' [star_targets] ')' | '[' [star_targets] ']'
 
-targets: target (',' target)* [',']
+targets[asdl_seq*]: a=','.target+ [','] { a }
```

Comment:
I predict a merge conflict on this line. :-)

Reply:
I have resolved this conflict locally, would it be okay to force-push this or should I do something else?

Reply:
Typically one would use `git merge` but if you've already rebased I am okay with a force push.

Reply:
I'll keep that in mind for the future.